pom.xml

```diff
@@ -57,6 +57,22 @@
 			<version>18.0</version>
 		</dependency>
 		<dependency>
+			<groupId>org.hamcrest</groupId>
+			<artifactId>hamcrest-core</artifactId>
+			<version>1.3</version>
```

Comment:
It's actually a dependency of junit so it gets pulled in anyways,  my apologies, will remove that explicit dep